modules/rdg/src/main.C

```diff
@@ -0,0 +1,29 @@
+#include "RdgApp.h"
+#include "MooseInit.h"
+#include "Moose.h"
+#include "MooseApp.h"
+#include "AppFactory.h"
+
+// Create a performance log
+PerfLog Moose::perf_log("Rdg");
+
+// Begin the main program.
+int main(int argc, char *argv[])
```

Comment:
Strictly speaking `int` should be on a separate line ;-) I guess we just did not modify `stork` yet...